requirements.txt

```diff
@@ -1,15 +1,16 @@
-configobj>=5.0.5
 cherrypy==17.3.0
-ws4py>=0.3.2
-SQLAlchemy>=1.1.0
-six>=1.5.2
+configobj>=5.0.5
 Jinja2>=2.7
-rpctools>=0.3.1
 logging_unterpolation>=0.2.0
-requests>=2.2.1
 paver>=1.2.2
-wheel>=0.24.0
 pip>=1.5.6
-sh>=1.09
-python-prctl>=1.6.1; 'linux' in sys_platform
 psutil>=5.4.1
+python-prctl>=1.6.1; 'linux' in sys_platform
+redis==4.3.6
```

Comment:
I sorted this file which makes this change unclear, but the only changed dependency is adding redis==4.3.6.